### PR TITLE
Updating russian localization

### PR DESCRIPTION
**Pull request by Kush1Push1**
## About The Pull Request

Updated russian localization to certain radio channels and some other chat's functions.

## Why It's Good For The Game

Keeping the code up-to date and fixes a bug for newly created ru-servers (lack of ability to say in radio without using english keyboard).

## Changelog
:cl: SishTis
fix: Updated russian keyboard localization. You can now use russian letter as prefix for radio channel, like, :у for engineering.
/:cl:

### CODE REVIEW

**Comment by Kush1Push1** [comment]
image

**Comment by bobbahbrown** [comment]
testing your code is based and i love people who test code good pr

**Comment by interestingusernam3** [comment]
Updated localization files.

File: code/modules/mob/living/living_say.dm [hunk 1]
```diff
@@ -29,32 +29,33 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 
 
 	//kinda localization -- rastaf0
-	//same keys as above, but on russian keyboard layout. This file uses cp1251 as encoding.
+	//same keys as above, but on russian keyboard layout.
+	//updated to UTF-8, 12.02.2021
```
Comment (Jared-Fogle) [inline code/modules/mob/living/living_say.dm:33]:
```suggestion
	//same keys as above, but on russian keyboard layout.
```

We don't need comments about what code used to be.

Comment (Kush1Push1) [reply code/modules/mob/living/living_say.dm:33]: Done.

**Comment by Low3** [comment]
based

**Comment by Stalkeros2** [comment]
:+1:

**Pull request merged**